workflows/chatbot/inference/backend/chat/tts.py

```diff
@@ -35,7 +35,7 @@ def __init__(self):
         )
         self.vocoder = SpeechT5HifiGan.from_pretrained("microsoft/speecht5_hifigan")
         self.vocoder.eval()
-        self.default_speaker_embedding = torch.load('speaker_embeddings/spk_embed_default.pt') # load the default speaker embedding
+        self.default_speaker_embedding = torch.load('backend/chat/speaker_embeddings/spk_embed_default.pt') # load the default speaker embedding
```

Comment:
I think relative path reference is not correct if you just run `run_ipex.sh` under this folder. You can refer to this https://github.com/intel/intel-extension-for-transformers/blob/main/neural_chat/pipeline/plugins/audio/tts.py#L57C22-L58C1 to see how the absolute path is made.

Reply:
I modified the readme as follows, which means run_ipex.sh cannot run under this folder directly:
```shell
cd intel-extension-for-transformers/workflows/chatbot/inference
nohup bash backend/chat/run_ipex.sh
```
This is because in the controller.py, the codes as follows is in the folder workflows/chatbot/inference/backend/llmcache
```shell
parser.add_argument(
        "--cache-chat-config-file", default="cache_config.yml", help="the cache config file"
    )
    parser.add_argument(
        "--cache-embedding-model-dir", default="./instructor-large", help="the cache embedding model directory"
```shell
    )